### DISCUSSION TITLE
WindIO Yamls

### DISCUSSION THREAD

**Discussion by Jamie7963** [OpenFAST/openfast#1439] [Model Integration]
Hi, is there integration between the WindIO yaml files and OpenFAST?

**Comment by jjonkman** [comment]
Dear @James-Ilosta,

Yes.  The WindIO parameterization can be used to generate OpenFAST input files through the use of the Wind Energy with Integrated Servo-control (WEIS) framework that combines multiple NREL-developed tools to enable design optimization of floating offshore wind turbines: https://github.com/WISDEM/WEIS.

Best regards,
